src/components/InputBox.tsx

```diff
@@ -0,0 +1,109 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+import PropTypes from 'prop-types';
+import SubmitArrow from '/assets/submit.svg';
+import { useRef, useState } from 'react';
+
+interface inputBoxType {
+  addMessage: (text: string) => void;
+}
+
+function InputBox({ addMessage }: inputBoxType) {
+  const [text, setText] = useState('');
```

Comment:
`useState`를 사용할 때에도 타입을 명시해 주면 좋을 것 같아요!
```suggestion
  const [text, setText] = useState<string>('');
```